src/sentry/static/sentry/app/views/projectInstall/newProject.jsx

```diff
@@ -13,9 +13,9 @@ const NewProject = ({organization}) => (
       <Content>
         <DocumentTitle title="Sentry" />
         <CreateProject
-          hasIssueAlertOptionsEnabled={organization.features.includes(
-            'new-project-issue-alert-options'
-          )}
+          hasIssueAlertOptionsEnabled={
```

Comment:
Are you planning on following-up with frontend analytics event tracking? We generally log exposure to the experiment with `logExperiment` and track events with `trackAnalyticsEvent`, etc.

Reply:
Yep. It'll be a separate PR!